coins.json

```diff
@@ -634,5 +634,16 @@
         "xpub": "xpub",
         "xprv": "xprv",
         "explorer": "https://explorer.monetaryunit.org/tx/"
+    },
+    {
+        "id": "bitshares",
+        "name": "Bitshares",
+        "symbol": "BTS",
+        "decimals": 5,
+        "blockchain": "EOS",
+        "derivationPath": "m/44'/308'/0'/0/0",
```

Comment:
can you update derivation path according to https://github.com/satoshilabs/slips/blob/master/slip-0048.md ?

Reply:
Sure. Should we also change the derivation paths of the previous graphene blockchains we added (Steem, EOS and Bravo) in a separate PR?

Reply:
Can you please let us know how we should change the derivation path according to slip 48? From what we see slip 48 is not currently supported by wallet-core.

Reply:
we have created an issue to track this: https://github.com/TrustWallet/wallet-core/issues/486, so you can skip it now, we will handle it

Reply:
Thank you